### ISSUE TITLE
Electrical - When batteries are OFF (or 00.0V) a few objects are still operable

### ISSUE THREAD

**Issue by alepouna** [flybywiresim/aircraft#7699]
### Aircraft Version

Experimental

### Build info

```json
{
    "built": "2023-01-08T23:19:25+00:00",
    "ref": "refs/heads/experimental",
    "sha": "7865832062508907453d0f04b1b2c774158dbcb2",
    "actor": "frankkopp",
    "event_name": "manual",
    "pretty_release_name": "experimental:78658320",
    "version": "v0.10.0-exp.7865832"
}
```

### Describe the bug

If you turn OFF both batteries, APU/GPU then a few objects (explained below each) are still operable. 

Preconditions: 
- Battery switches both OFF or 00 volts (from draining)
- APU/GPU off/disconnected. 

![image](https://user-images.githubusercontent.com/98479040/211363042-2b9272ad-f8fa-4fa6-b73c-6d667ea2e884.png)
(In the screenshot I drained the batteries AND have the OFF switches off. I powered the airplane for a little bit to charge the batteries and its still replicable - also replicable after just loading in the sim) 

## ANN LT on TEST 

When putting the panel lights on TEST mode the ADIRS indicator lights up
![image](https://user-images.githubusercontent.com/98479040/211362593-2b008edb-eb5d-4a00-a5c8-d52906f950ea.png)
![image](https://user-images.githubusercontent.com/98479040/211362845-fc536d56-f298-442d-89bf-25c0308b07e6.png)

The batteries also indicators also show up as in TEST mode, and the AVAIL light of GPU but this could be normal behaviour? But without any power maybe not. Dunno!
![image](https://user-images.githubusercontent.com/98479040/211362979-9dfb936b-809b-4470-8018-d88a1fa12998.png)

## No Smoking switch activates audio queue 

If you interact with the No Smoking switch, you can hear the audio alert (and I want to assume the entire function operates as well) 
https://user-images.githubusercontent.com/98479040/211363285-ef8010b6-ccf6-4ba1-92af-7780eb584381.mp4
(Might need to turn the audio up, sorry!) 

## Evac horn plays the sound

https://user-images.githubusercontent.com/98479040/211363792-fc944808-4dda-4219-9746-e4d1a4571186.mp4
(Might need to turn the audio up, sorry!) 

### Discord Username (optional)

Alepouna🌙#9824

**Comment by auroraisluna** [comment]
To add a quick note, it has come to my attention that NAV lights behaviour is normal due to the airplane having a GPU avail. I am doing some other tests currently on the airplane, will confirm shortly by editing both OG post and posting a comment here :)

**Comment by auroraisluna** [comment]
Yup, seems to be as such. When GPU isn't AVAIL the NAV lights do not turn on either. Removed from post. Everything is replicable.

**Comment by auroraisluna** [comment]
Also I'd like to add, as someone mentioned on Discord (sorry whoever it was i can't find your message D:) that the backlight on the battery indications, could be off when there is no battery. More information would be very welcome about this as well.

**Comment by MicahBCode** [comment]
Something to add: Dome light is also working with both battery switches in OFF position and no EXT PWR conntected or APU running. Not sure if thats simulated correctly.

**Comment by tracernz** [comment]
> Something to add: Dome light is also working with both battery switches in OFF position and no EXT PWR conntected or APU running. Not sure if thats simulated correctly.

Can't replicate this on master.

Captain's dome light is powered from ground service bus, so it is on whenever external power is connected (doesn't need to be on), and FO's dome light is off one of the batt busses, I forget which, but it's the only light you have in emergency config (used to be SOP to have it on during takeoff). This is all correctly implemented.

**Comment by MicahBCode** [comment]
@tracernz Yeah with "no EXT PWR connected" i meant it is available ("AVAIL" is shown in the overhead) but not switched on. But as you stated this is in that case correctly implemented then.

**Comment by joeherwig** [comment]
Just a remark regarding the annunciators without batteries off.
I notived the the Simvars for instance for `FUELSYSTEM PUMP ACTIVE:1 etc. can still be changed even though there is no electricity connected at all.

If i want to rely within my homecockpit for the korry-annunciators on those simvars, they are still at the "on" value, even though all power is off and disconnected and within the VC the annunciators are off.

Probably it makes sense to provide the annunciator lights on LVars to make the values visible in the VC available externally.
Based on Simvars (A-Vars) that doesn't work right at the moment.
